api/configDB/config.ts

```diff
@@ -1,8 +1,22 @@
-import { Pool } from 'pg';
-
-const pool = new Pool({
-  connectionString: process.env.DATABASE_URI,
-  ssl: { rejectUnauthorized: false },
-});
-
-export default pool;
+import { Pool } from 'pg';
+import * as winston from 'winston';
+
+const pool = new Pool({
+  connectionString: process.env.DATABASE_URI,
+  ssl: { rejectUnauthorized: false },
+});
+
+async function connectWithTheDatabase() {
+  await pool

```

Comment:
Może bardziej w ten sposób: 
try {
await pool.connect()
winston.info('Connected with the Database');
} catch {
...
}

Reply:
A ok, poprawie potem, właśnie nie do końca zrozumiałem wcześniej :)